buddy_mentorship/models.py

```diff
@@ -54,20 +55,26 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
         request_type_str = ["Request", "Offer"][int(self.request_type)]
+        requestor_profile = Profile.objects.get(user=self.requestor)
+        requestee_profile = Profile.objects.get(user=self.requestee)
 
         def email_context(self):
             return {
                 "request_type_str": request_type_str,
                 "requestor": self.requestor,
                 "requestee": self.requestee,
                 "message": self.message,
-                "profile_url": reverse("profile", args=[profile.id]),
+                "requestor_profile_url": reverse(
+                    "profile", args=[requestor_profile.id]
+                ),
+                "requestee_profile_url": reverse(
+                    "profile", args=[requestee_profile.id]
+                ),
                 "request_detail_url": reverse("request_detail", args=[self.id]),
                 "APP_URL": os.getenv("APP_URL"),
             }
 
         if self.status == 0:
```

Comment:
non blocking but could we change these to check against the static named variables instead of numbers?